### DISCUSSION TITLE
set object policy/acl during upload

### DISCUSSION THREAD

**Discussion by till** [goreleaser/goreleaser#3129] [Ideas, Issue Triage, and General discussions]
Our s3-compatible storage does not support bucket policies (like AWS or Minio do). So instead, we upload files and set a header to make the file public. For example, if I was to use `mc` (or any AWS SDK), I set an additional header of `x-amz-acl: public-read` with the request and it works.

I dug into the code briefly, it would probably need to go there:
https://github.com/goreleaser/goreleaser/blob/df0216d5855e9283d2106fb5acdb0e7b528a56e8/internal/pipe/blob/upload.go#L229-L231

I would expose it via the config for `blobs` in `.goreleaser.yml`.

The questions are:

1. Would you be willing to support this in go releaser? 
2. And then, any thoughts in general — e.g. config, or am I looking at the correct place?

I am happy to come up with a PR for this.

**Comment by caarlos0** [comment]
this is being discussed https://github.com/goreleaser/goreleaser/issues/3069
